lib/doge/DogeRpc.js

```diff
@@ -0,0 +1,10 @@
+const BtcRpc = require('../btc/BtcRpc');
+class DogeRpc extends BtcRpc {
+  async estimateFee() {
```

Comment:
It appears that doge has the 'estimateSmartFee' method, so maybe we don't need this